### PR TITLE
sig-k8s: Update member-list.md

### PR DESCRIPTION
**Pull request by cofyc**
I sort the list by contributions, so the order may change.

### CODE REVIEW

File: special-interest-groups/sig-k8s/member-list.md [hunk 1]
```diff
@@ -39,25 +39,30 @@ None
 - [shonge](https://github.com/shonge)
 - [cwen0](https://github.com/cwen0)
 - [mikechengwei](https://github.com/mikechengwei)
-- [shinnosuke-okada](https://github.com/shinnosuke-okada)
+- [kolbe](https://github.com/kolbe)
+- [sokada1221](https://github.com/sokada1221)
```
Comment (cofyc) [inline special-interest-groups/sig-k8s/member-list.md:43]: `shinnosuke-okada` changed the nickname to `sokada1221`

**Comment by cofyc** [comment]
@AndreMouche in addition to adding new members, I pushed a new commit to update the format following the conventions in `sig-template`

File: special-interest-groups/sig-k8s/README.md [hunk 2]
```diff
@@ -18,10 +19,15 @@ See [SIG K8s Member List](./member-list.md)
 
 ## Contact
 
-* Slack: channel #sig-k8s in the [tidbcommunity](https://pingcap.com/tidbslack) slack workspace.
+* Slack: [#sig-k8s](https://tidbcommunity.slack.com/archives/CHD0HA3LZ) in the [TiDB Community](https://pingcap.com/tidbslack) workspace
+
+## Repositories, Projects and labels
+
+- [tidb-operator](https://github.com/pingcap/tidb-operator)
```
Comment (DanielZhangQD) [inline special-interest-groups/sig-k8s/README.md:26]: Should we add `docs-tidb-operator`?

Comment (cofyc) [reply special-interest-groups/sig-k8s/README.md:26]: no, all docs repos belong to `sig-docs`

**Pull request merged**